src/higher-order.js

```diff
@@ -152,6 +153,11 @@ function branch(cursors, Component) {
       this.watcher.refresh(mapping);
       this.setState(this.watcher.get());
     }
+
+    // Should update the component?
+    shouldComponentUpdate(nextProps, nextState) {
+      return !(deepEqual(this.props, nextProps, {strict: true}) && deepEqual(this.state, nextState, {strict: true}))
```

Comment:
@kakadiya91 What's the performance difference of using `deepEqual` over just `===` on every prop?

I'd have thought that the `===` would give us most of the benefit (and the `deep-equal` module looks like a surprisingly large amount of code).